etc/config-template.toml

```diff
@@ -43,6 +43,11 @@
 # set attributes about this server, e.g. { zone = "us-west-1", disk = "ssd" }.
 # labels = {}
 
+# set the path for certificates. Empty string means disabling secure connectoins.
+# ca-path = ""
```

Comment:
Maybe we need to support different certificate to communicate with PD.

Reply:
Actually, this ca is just for the communications between TiKV, see https://github.com/pingcap/tikv/pull/2506/files#diff-fad2b574d6af9e2e06ebcfffb16cc4f7R25

Reply:
For now the client has the ability to verify the pd server identity and provide transport security. We also need the support of client certificates to prevent unauthorized access to pd server, so client in tikv should specify client-cert-path and specify client-key-path